spec/app_config_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative '../lib/app_config'
+require 'pp'
+
+
+describe AppConfig do
```

Comment:
- Try to avoid `should` at the start of the sentence. `it "loads the config"` reads better and avoids repetition.